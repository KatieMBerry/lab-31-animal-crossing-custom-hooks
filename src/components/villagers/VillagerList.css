.VillagerList {
    display: flexbox;
    flex-wrap: wrap;
  }

  img {
      width: 5rem;
      height: 7rem;
  }
  
  