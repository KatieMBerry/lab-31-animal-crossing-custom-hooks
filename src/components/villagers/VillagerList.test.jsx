import React from 'react';
import { render, cleanup } from '@testing-library/react';
import VillagerList from './Villagers.List';

describe('VillagerList component', () => {
    afterEach(() => cleanup());
    it('renders VillagerList', () => {
        const { asFragment } = render(<VillagerList
            villagers={[]}
        />);
        expect(asFragment()).toMatchSnapshot();
    });
});
