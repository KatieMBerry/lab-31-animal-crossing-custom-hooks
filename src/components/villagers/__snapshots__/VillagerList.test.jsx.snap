// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VillagerList component renders VillagerList 1`] = `
<DocumentFragment>
  <ul
    data-testid="villagers"
  />
</DocumentFragment>
`;
